pkg/geo/geomfn/binary_predicates_bench_test.go

```diff
@@ -0,0 +1,62 @@
+		geo.MustParseGeometry("MULTIPOLYGON (((-109.212025519999941 10.302679755000042, -109.210357225999928 10.288723049000055, -109.218576626999948 10.28156159100007, -109.228016730999911 10.292141018000052, -109.234242316999939 10.307928778000075, -109.225941535999937 10.311021226000037, -109.212025519999941 10.302679755000042)))"),
+	}
+	benchPoints = []geo.Geometry{
+		geo.MustParseGeometry("POINT (19.383354043259374 -29.13291298545164)"),
+		geo.MustParseGeometry("POINT (15.547774209259558 45.487209288609961)"),
+		geo.MustParseGeometry("POINT (10.767294561862002 35.730702135151205)"),
+		geo.MustParseGeometry("POINT (24.540041868800813 -32.300006491323757)"),
+		geo.MustParseGeometry("POINT (94.116570189391609 25.666997903439437)"),
+		geo.MustParseGeometry("POINT (-65.929991497446565 -17.589999980848233)"),
+		geo.MustParseGeometry("POINT (1.22081126074562 6.133882930268385)"),
+		geo.MustParseGeometry("POINT (-85.860041139453415 11.249996764058892)"),
+		geo.MustParseGeometry("POINT (27.449993447588497 -7.300033753593539)"),
+		geo.MustParseGeometry("POINT (-73.340004314859073 -37.480000387828738)"),
+	}
+)
+
+var benchRes bool
+
+func BenchmarkIntersects(b *testing.B) {
```

Comment:
let's rename this `BenchmarkPointAndLargePolygonIntersect`

i wouldn't mind adding another mini benchmark for smaller polygons / points, so we have at least two data sets that are interesting!